lib/created_message_queue_handler.ts

```diff
@@ -101,6 +103,33 @@ const messageContainerName = getRequiredStringEnv("MESSAGE_CONTAINER_NAME");
 const storageConnectionString = getRequiredStringEnv("QueueStorageConnection");
 const queueConnectionString = getRequiredStringEnv("QueueStorageConnection");
 
+const documentClient = new DocumentDBClient(cosmosDbUri, {
```

Comment:
I've moved the inizialization code for models outside index() as 
1. this is more coherent on what we do in the api handler (public_api_v1.ts)
2. this will eventually fail at deploy time (not a runtime)
3. it's impossible to handle this kind of errors with retries (as we need the models and services to be valid)

Reply:
I don't understand reason n.2, what do you mean? (also put these reasons as comments in code, e.g. we're initializing the db client here because...)

Reply:
reason 2 is wrong (I thought that the function runtime would require the module at startup but it doesn't)